uppsrc/CtrlCore/X11Proc.cpp

```diff
@@ -418,17 +420,20 @@ void Ctrl::EventProc(XWindow& w, XEvent *event)
 		}
 		break;
 	case MotionNotify:
-		while(XCheckWindowEvent(Xdisplay, top->window, PointerMotionMask, event));
-		EndIgnore();
-		mousePos = Point(event->xmotion.x_root, event->xmotion.y_root);
-		Xeventtime = event->xmotion.time;
-		Point p = mousePos - Xbuttonpos;
-		if(max(abs(p.x), abs(p.y)) > 4)
-			Xbuttontime = Xeventtime - 0x80000000;
-		sModState = event->xmotion.state;
-		DispatchMouse(MOUSEMOVE, Point(event->xmotion.x, event->xmotion.y));
-		DoCursorShape();
-		break;
+		Top *top = GetTop(); //aris002
+		if(top) {
```

Comment:
Remove this line:
```
`Top *top = GetTop(); //aris002`
and replace with
if (!top) {
    break;
}

/// refer to utop variable instead of top...
```

Reply:
Same as https://github.com/ultimatepp/ultimatepp/pull/126#discussion_r1059700681 .  But  please check in how often is this line (Top *top = GetTop();)  used in other CtrlCore places...  But my main question still : X11 Window might exist, therefore utop==true but  top==false?  or what about topmost?